### ISSUE TITLE
☂️  Desktop accessibility support

### ISSUE THREAD

**Issue by HansMuller** [flutter/flutter#73819]
This is an umbrella issue that covers the work being done to support accessibility for desktop Flutter applications.

### Design documents

- go/flutter-desktop-accessibility
- go/accessibility-bridge-detailed-design

### Related issues

- https://github.com/flutter/flutter/issues/23601

### Flutter Engine Changes

- [x] Forked chromium accessibility code https://github.com/flutter/engine/pull/22909
- [x] Initial import of accessibility code from Chromium https://github.com/flutter/engine/pull/23087
- [x] Apply local patch to chromium accessibility code https://github.com/flutter/engine/pull/23110
- [x] Implements accessibility bridge in common library https://github.com/flutter/engine/pull/23491

mac:
- [x] Wires up accessibility bridge in macOS https://github.com/flutter/engine/pull/23870
- [x] selection announcement does not work for selectable text https://github.com/flutter/flutter/issues/77833
- [x] voice over does not announce feedback when typing in a text field https://github.com/flutter/flutter/issues/77834
- [x] text field should be activated when it receives a11y focus https://github.com/flutter/flutter/issues/77835

windows:
- [x] Forked required windows component from chromium https://github.com/flutter/flutter/issues/77836
- [ ] Wires up accessibility bridge in Windows https://github.com/flutter/flutter/issues/77838

### Chromium Changes

- https://chromium-review.googlesource.com/c/chromium/src/+/2616901

**Comment by chunhtai** [comment]
cc @gw280
